Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [LOOT/Prelude](https://hosted.weblate.org/projects/loot/prelude/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/loot/prelude/horizontal-auto.svg)

Commit:
Translated using Weblate (Spanish)

Currently translated at 79.6% (51 of 64 strings)

Translation: LOOT/Prelude
Translate-URL: https://hosted.weblate.org/projects/loot/prelude/es/

Ortham commented:
I didn't actually make any changes in Weblate, all I did was approve the existing translations, so I think this is just happening because Weblate has wrapped the lines.

I've now installed the "Customise YAML output" add-on and configured it to use:

- Indentation of 2 spaces
- No line wrapping
- \n line endings

Ortham commented:
Just noting that this ties back to loot/loot#1973.

weblate committed:
Translated using Weblate (Bulgarian)

Currently translated at 93.7% (60 of 64 strings)

Translation: LOOT/Prelude
Translate-URL: https://hosted.weblate.org/projects/loot/prelude/bg/

weblate committed:
Translated using Weblate (Danish)

Currently translated at 82.8% (53 of 64 strings)

Translation: LOOT/Prelude
Translate-URL: https://hosted.weblate.org/projects/loot/prelude/da/

weblate committed:
Translated using Weblate (Spanish)

Currently translated at 79.6% (51 of 64 strings)

Translation: LOOT/Prelude
Translate-URL: https://hosted.weblate.org/projects/loot/prelude/es/

weblate committed:
Translated using Weblate (French)

Currently translated at 90.6% (58 of 64 strings)

Translation: LOOT/Prelude
Translate-URL: https://hosted.weblate.org/projects/loot/prelude/fr/

weblate committed:
Translated using Weblate (Finnish)

Currently translated at 92.1% (59 of 64 strings)

Translation: LOOT/Prelude
Translate-URL: https://hosted.weblate.org/projects/loot/prelude/fi/

weblate committed:
Translated using Weblate (Italian)

Currently translated at 92.1% (59 of 64 strings)

Translation: LOOT/Prelude
Translate-URL: https://hosted.weblate.org/projects/loot/prelude/it/

weblate committed:
Translated using Weblate (Korean)

Currently translated at 78.1% (50 of 64 strings)

Translation: LOOT/Prelude
Translate-URL: https://hosted.weblate.org/projects/loot/prelude/ko/

weblate committed:
Translated using Weblate (Japanese)

Currently translated at 90.6% (58 of 64 strings)

Translation: LOOT/Prelude
Translate-URL: https://hosted.weblate.org/projects/loot/prelude/ja/

weblate committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 92.1% (59 of 64 strings)

Translation: LOOT/Prelude
Translate-URL: https://hosted.weblate.org/projects/loot/prelude/pt_BR/

weblate committed:
Translated using Weblate (Portuguese (Portugal))

Currently translated at 78.1% (50 of 64 strings)

Translation: LOOT/Prelude
Translate-URL: https://hosted.weblate.org/projects/loot/prelude/pt_PT/

weblate committed:
Translated using Weblate (Polish)

Currently translated at 84.3% (54 of 64 strings)

Translation: LOOT/Prelude
Translate-URL: https://hosted.weblate.org/projects/loot/prelude/pl/

weblate committed:
Translated using Weblate (Chinese (Simplified))

Currently translated at 78.1% (50 of 64 strings)

Translation: LOOT/Prelude
Translate-URL: https://hosted.weblate.org/projects/loot/prelude/zh_Hans/

weblate committed:
Translated using Weblate (Ukrainian)

Currently translated at 93.7% (60 of 64 strings)

Translation: LOOT/Prelude
Translate-URL: https://hosted.weblate.org/projects/loot/prelude/uk/

weblate committed:
Translated using Weblate (Russian)

Currently translated at 93.7% (60 of 64 strings)

Translation: LOOT/Prelude
Translate-URL: https://hosted.weblate.org/projects/loot/prelude/ru/

weblate committed:
Translated using Weblate (Turkish)

Currently translated at 93.7% (60 of 64 strings)

Translation: LOOT/Prelude
Translate-URL: https://hosted.weblate.org/projects/loot/prelude/tr/

weblate committed:
Translated using Weblate (Swedish)

Currently translated at 67.1% (43 of 64 strings)

Translation: LOOT/Prelude
Translate-URL: https://hosted.weblate.org/projects/loot/prelude/sv/
